docs/online/src/scenarios/postmessage.rst

```diff
@@ -269,6 +270,76 @@ You can send the following messages; all others are ignored:
         }
 
 
+..  data:: Host_RefreshSessionInfo
+
+    When the host wants to send new tokens to |wac|, for instance when an :term:`access token` is about to expire, 
+    a :data:`Host_RefreshSessionInfo` message is posted. The host can send a set of new access tokens and their properties 
+    in this post message. 
```

Comment:
Can you explain the flow? Does the host decide when to refresh tokens, or does the app? Or both? Is the app expected to respond to these messages all the time?